src/deploy/deploy.py

```diff
@@ -154,8 +154,10 @@ def _deploy(project, release, environment_id, description, confirm=True):
     click.echo(click.style(f"Requested by: {release['requested_by']}", fg="yellow"))
     click.echo(click.style(f"Date created: {release['date_created']}", fg="yellow"))
 
-    ecs_service_arns = project.get_ecs_service_arns(
+    ecs_service_arns = ecs.find_service_arns_for_release(
+        project=project._underlying,
         release=release,
+        service_descriptions=project.ecs._described_services,
```

Comment:
Not sure what the difference between `project._underlying` and `project` are, but could the `service_descriptions` just be got from the `project` argument?

Reply:
This is the old class/new class thing.

* `project` is the old `Project` class from `project.py`
* `project._underlying` is the new `Project` class from `models.py`

The plan is that eventually, there will only be the new class (which is a precise representation of the contents of `.wellcome_project`), and every reference to `project._underlying` will become `project` – but I'm refactoring that way a bit at a time.